### PR TITLE
Split votes page

### LINKED ISSUE #320: view for close / split votes
Once #318 comes in, it would be useful to have a view highlighting close / split votes in City Countil. 

The `votecount` table makes this pretty easy to get at. Here's all vote counts between 15 and 35 votes: https://puddle.bunkum.us/chicago_council-adf4eb4/votecount?_sort=id&value__lte=35&value__gte=15

One option would be to create a new page that lists all close votes, ordered by date.

Another option would be to include these vote counts as filters in the legislation search. This would be more useful as you could combine other search filters, though it would require adding the vote counts, which happen at the bill action level, to the search index

### PR DESCRIPTION
**Pull request by derekeder**
Creates a new Split votes / dissenting votes page.

This page displays a list of legislation in a table where there are recorded 'no' votes. Currently, it is showing all legislation with any no votes and yields 700+ bills. Setting this threshold higher to 10 votes shows 100 bills, which is maybe more interesting and certainly loads the page faster.

Screen Shot 2023-01-12 at 10 53 27 PM

I found it useful to refer to one of Dick Simpson's reports on how to present this information. We may want to do some similar things: https://www.chicagotribune.com/opinion/commentary/ct-opinion-lightfoot-two-years-city-council-simpson-20210519-ogs6eifmuzghrevydirv63zwwe-story.html

closes #320

### CODE REVIEW

**Review by fgregg — CHANGES_REQUESTED** [review]
Love where this is going. A few high level points.

1. I think it would make sense to segment by legislative session, i.e. term.
2. It would be good to add links for the vote to the detailed information about the vote on the bill page. We should be able to construct a slug with the name of the action and the date that can be used as anchor tags.
3. There can be multiple split votes on the same bill, i.e. in committee and then also on the final action. I think it would be great to show all those, maybe with a table in a table view. (there's seldom going to be more than three or so). 

For the future, it would be good to keep track of the mayor's floor leader so we can know whether the vote was going for or against the mayor.

@red-bin has been looking at some similar things.

File: chicago/views.py [hunk 1]
```diff
@@ -294,6 +294,24 @@ def substitute_ordinance_redirect(request, substitute_ordinance_slug):
...
 
 
+class ChicagoSplitVotesView(ListView):
+    template_name = "split_votes.html"
+    context_object_name = "bills"
+
+    def get_queryset(self):
+        return (
+            ChicagoBill.objects.filter(
+                actions__vote__counts__value__gt=0, actions__vote__counts__option="no"
```
[the remark this answers is not in the document]
Comment (derekeder) [reply chicago/views.py:304]: good call. i'll filter on just the full council

**Comment by derekeder** [comment]
Notes on things to do:

- use django `url` for legislation links
- note who the mayor is for each legislative session
- remove 2007 legislation session (little/no data)

File: chicago/templates/council_members.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h4>Look up your council member</h4>
 
 
 {% block extra_js %}
-  {% cache 86400 js_wrapper 'js' %}
+  {% cache 86400 js_wrapper_compare 'js' %}
```
Comment (derekeder) [inline chicago/templates/council_members.html:50]: using the same cache wrapper on the council members map and compare page is causing an error

**Comment by derekeder** [comment]
@fgregg ok this is ready for another look

- filtering on legislative session
- more efficient vote query
- adds nav link
- showing who the mayor is for each session
- removed 2007 from session list
- handling not found legislative session 404

Unrelated, adds a fix for the broken Aldermen map on production.

Screen Shot 2023-01-13 at 4 45 55 PM

**Review by fgregg — APPROVED** [review]

**Pull request merged**